R/tar-terra-rast.R

```diff
@@ -32,6 +32,20 @@
 #'   are retained by archiving all written files as a zip file upon writing and
 #'   unzipping them upon reading. This adds extra overhead and will slow
 #'   pipelines.
+#' @param preserve_metadata character. When `"drop"` (default), any
+#'   auxiliary files that would be written by [terra::writeRaster()] containing
+#'   raster metadata such as units and datetimes are lost (note that this does
+#'   not include layer names set with `names() <-`).  When `"zip"`, these
+#'   metadata are retained by archiving all written files as a zip file upon
+#'   writing and unzipping them upon reading. This adds extra overhead and will
+#'   slow pipelines. Also note metadata may be impacted by different versions
+#'   of GDAL and different drivers. If you have an issue with retaining
+#'   metadata for your setup, please file an issue at
+#'   \url{https://github.com/ropensci/geotargets/issues/} and we will try and
```

Comment:
This makes me think we should eventually create an issue template asking users to try a reprex with just `terra`.  We've seen a few issues opened in `geotargets` that are just `terra` behavior that we can't do much about.

Reply:
Agreed, an issue template would be good